packages/agent/conf/examples/templates/kvk/KVKRegistration.hbs

```diff
@@ -0,0 +1,12 @@
+{
+    "@context": [
+        "https://www.w3.org/2018/credentials/v1"
+    ],
+    "type": [
+        "VerifiableCredential",
+        "KVKRegistration"
+    ],
+    "description": "This credential proves that the company you represent is registered with KVK",
```

Comment:
Description should be in the credential subject and made shorter. We need to be able to display it.

"Company registered with the KvK"

Reply:
done